### DISCUSSION TITLE
Delete One Item

### DISCUSSION THREAD

**Discussion by maxoakland** [gingerbeardman/stapler#30] [Q&A] [answered]
If you have a question, please use Discussions (link above)

**Is your feature request related to a problem? Please describe.**
Right now if you hit delete, it deletes all items in your list.

**Describe the solution you'd like**
 I'd like to just be able to delete one item

**Describe alternatives you've considered**
You could have it in a toolbar

**Comment by gingerbeardman** [comment, marked as answer]
Select a single item (by clicking, or using cursor keys) then press delete (key or menu item)

https://github.com/user-attachments/assets/c0092f38-308d-409b-87a2-091d9303b3ab
